rolling/distribution.yaml

```diff
@@ -3107,7 +3107,7 @@ repositories:
       tags:
         release: release/rolling/{package}/{version}
       url: https://github.com/ros2-gbp/rmf_api_msgs-release.git
-      version: 0.0.1-1
+      version: 0.0.1-3
```

Comment:
did you forget to bump the patch number?
or did you intentionally only bump the revision intentionally?

Reply:
only bumping the revision here, thanks for the double check.